src/compose/ComposeMenu.js

```diff
@@ -19,6 +19,26 @@ type Props = {
   onExpandContract: () => void,
 };
 
+/**
+ * In 2017, Apple adopted a new standard image format, HEIF, which stands
+ * for High Efficiency File Format; Apple photos are named with the '.heic'
+ * extension. When an image file with the extension '.heic' is uploaded to the
+ * server,the image becomes unviewable within a message list. Interestingly,
+ * the `file` command line utility reports that images saved under this format
+ * are JPEG files, so there's likely some pecularity with how the server handles
+ * file uploads based on the extension.
+ *
+ * If an image file name has the '.heic' extension, and the uri of the image on a
+ * mobile device has an extension for the JPEG format, fixFileNameFromUri replaces
+ * that extension with '.jpeg'.
+ */
+export const fixFileNameFromUri = (uri: string, fileName: string): string => {
```

Comment:
The name of the function can be made better. 'Fix' is a bit generic. 'FileName' might not be exact (we are talking about extension only). Would also adding 'image' somewhere in the name help? (I know I am a bit vague, but trying not to suggest you a concrete name :) )

I would also move this to some of the `/utils/*.js` files.

Reply:
Yeah, keeping it in the same directory but outside the `ComposeMenu` makes sense. We don't use that approach often right now, but it is perfectly valid and we might want to use it more frequently. 👍

Reply:
I could have been more specific; I'm ok with keeping the function in the `ComposeMenu` file.